index.js

```diff
@@ -29,20 +33,104 @@ cp.parseBlock = function () {
 	return this.finishNode(node, "BlockStatement");
 };
 
+function strToModuleIdentifier(str) {
+	return '$m_' + str.replace(/-(\w)/g, (_, letter) => letter.toUpperCase()).replace(/[^$_a-zA-Z0-9]/g, '_');
+}
+
+function writePrefix (imports) {
+	let transformed = `Object.defineProperty(exports, '__esModule', { value: true });\n`;
```

Comment:
@jdalton They will have to do this in browsers anyway (with `<script type="module">` vs regular `<script>`), why not reflect same here?

Also, the only case where it will really make difference is that one will need to write e.g. `import * as fs from 'fs'` instead of `import fs from 'fs'` - should be acceptable IMO.